src/types.ts

```diff
@@ -7,18 +7,21 @@ export type SUPPORTED_NETWORKS =
   | "ethereum"
   | "opbnbtestnet"
   | "opbnbmainnet"
-  | "zksyncsepolia";
+  | "zksyncsepolia"
+  | "opsepolia"
+  | "opmainnet";
 
 export type REMOTE_NETWORKS =
   | "sepolia"
   | "ethereum"
   | "opbnbtestnet"
   | "opbnbmainnet"
   | "arbitrumsepolia"
-  | "arbitrumone";
+  | "arbitrumone"
+  | "opsepolia";
```

Comment:
```suggestion
  | "opsepolia"
  | "opmainnet";
```

Reply:
I have not added the configurations for `opmainnet` and due to which checks will fail if I add `opmainnet` to `REMOTE_NETWORKS`.